subsys/bootloader/bl_validation/bl_validation.c

```diff
@@ -8,6 +8,28 @@
 #include <zephyr/types.h>
 #include <fw_info.h>
 #include <kernel.h>
+#include <bl_storage.h>
+
+
+int set_monotonic_version(u16_t version, u16_t slot)
+{
+	__ASSERT(version <= 0x7FFE, "version is too large for slot to fit. "
+		"Final number cannot be 0xFFFF.\r\n");
+	__ASSERT(slot <= 1, "Slot must be either 0 or 1.\r\n");
+	printk("Setting monotonic counter (version: %d, slot: %d)\r\n",
+		version, slot);
+	return set_monotonic_counter((version << 1) | !slot);
```

Comment:
Everyone: Note that I changed this from `slot` to `!slot` to ensure that slot 0 takes precedence over slot 1.